docs/Rules.md

```diff
@@ -82,3 +83,7 @@ Syntax/parsing errors that appear during the linting process are reported with t
 ## ui5-class-declaration
 
 Checks whether the declaration of UI5 classes is correct. This rule only applies to TypeScript code where built-in ECMAScript classes are used instead of an `.extend()` call.
+
+## unsupported-api-usage
+
+Checks whether used syntax is unsupported by UI5.
```

Comment:
```suggestion
Checks whether UI5 API is used correctly, for example in a binding declaration a formatter has to be of type `function` when it's declared in a JavaScript/TypeScript file.
```

Reply:
@klattg please verify

Reply:
```suggestion
Checks whether the UI5 API is used correctly, for example, whether a formatter in a JavaScript/TypeScript binding declaration is of type `function`.
```

Reply:
@flovogt I'd suggest to write it like this. Is it OK for you?

Reply:
LGTM